### ISSUE TITLE
Email: Date/time picker defaults to unexpected timezone

### ISSUE THREAD

**Issue by richardolsson** [zetkin/app.zetkin.org#1810]
## Description
In the new work in progress email feature, the scheduling UI uses a new time picker component that includes timezone. In my experience, that picker defaults to GMT+9, which is unexpected.

## Steps to reproduce
1. Go to /organize/1/projects (currently on the epic/mass-email branch)
2. Click on Create to add new email
3. In the new email, quickly configure a targeting smart search, lock the targeting, add some content in the compose/design tab
4. Click the "delivery" button in the top right

### Expected Behaviour
The time picker that appears should default to the timezone of the system/browser

### Actual Behaviour
The time picker defaults to GMT+9

## Screenshots
![bild](https://github.com/zetkin/app.zetkin.org/assets/58265097/f695236c-9402-4b10-8b1e-b1bf47fbf41c)

**Comment by richardolsson** [comment]
Thank you @djbusstop! You're right about the hard-coded value, I wasn't aware of it when I found this "bug". It turned out it had been hard-coded due to some problem on the backend, and that has now been fixed (but not deployed yet). So it should be possible soon to remove the hard-coded value, and then make sure that everything works as expected. I'm not sure it will, and it sounds like you also found some potential problems, but it's blocked for now.
